openshift/jenkins/Jenkinsfile

```diff
@@ -0,0 +1,323 @@
+#!groovy
+
+// ------------------
+// Pipeline Variables
+// ------------------
+def cli
+def notify
+
+// --------------------
+// Declarative Pipeline
+// --------------------
+pipeline {
+  agent none
+  options {
+    disableResume()
+    buildDiscarder(logRotator(numToKeepStr: "5")) // keep 5 builds only
+  }
+  environment {
```

Comment:
I couldn't get environment vars to work for some reason.  How did you get it to work?

Reply:
They just work? 😉 
I did have issues with **boolean** values so I've used string values instead without problems